src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -36,10 +36,10 @@
   string via (apply str v)."
```

Comment:
No strong opinion - was just being a bit conservative (and suppose there's also the clojure precedents like let).